### DISCUSSION TITLE
Datahub graphql Allow all columns of an entity

### DISCUSSION THREAD

**Discussion by mhstudioos** [pimcore/pimcore#13721] [Q&A / Support] [answered]
Hi

Not sure if this is already possible but couldn't find proper docs about it.

Is it possible to allow all collumns/properties of an entity in the datahub without selecting them all via the interface?

**Comment by CodeBrauer** [comment, marked as answer]
I don't think this is currently possible using the UI.
Either create a custom query schema: https://pimcore.com/docs/data-hub/current/GraphQL/Query/Add_Custom_Query.html
Or you can edit the Datahub .yaml file in `var/config/data-hub/`.
